### PR TITLE
Version two

### PR DESCRIPTION
**Pull request by Drishtantr**
Created a test section, added instructions pop up in the test page, made a static results page, added mkbvue for graphs, vuesax and material-icons for styling and alerts

### CODE REVIEW

File: src/views/Test.vue [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    countDownChanged(dismissCountDown) {
+      this.dismissCountDown = dismissCountDown;
+    },
+    showAlert() {
+      if (this.successAlert) {
+        this.dismissCountDown = this.dismissSecs;
+      }
+    },
+    async addItem() {
+      // eslint-disable-next-line no-restricted-globals
```
Comment (shakyaprasen) [inline src/views/Test.vue:60]: isntead of this, you can add window to eslint-globals in eslint config file and use window.confirm

File: src/views/Test.vue [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+          // this.newTitle = '';
+          // this.current_user_email = '';
+          this.successAlert = true;
+          this.acceptAlert();
+          this.$router.push({ name: 'Result' });
+        }
+      }
+    },
+    // eslint-disable-next-line no-unused-vars
+    acceptAlert(color) {
```
Comment (shakyaprasen) [inline src/views/Test.vue:82]: no need to disable unused vars just use the color from argument on line 84

File: src/views/Transcriptions.vue [hunk 3]
```diff
@@ -78,7 +59,15 @@ export default {
     deleteItem(id) {
       db.collection('transData').doc(id).delete();
       this.deleteAlert = true;
-      this.showAlert();
+      this.acceptAlert();
+    },
+    // eslint-disable-next-line no-unused-vars
+    acceptAlert(color) {
+      this.$vs.notify({
+        color: 'danger',
```
Comment (shakyaprasen) [inline src/views/Transcriptions.vue:67]: same as above

File: src/views/Upload.vue [hunk 4]
```diff
@@ -81,9 +61,18 @@ export default {
         this.audioLength = '';
         this.newComment = '';
         this.successAlert = true;
-        this.showAlert();
+        this.acceptAlert();
+        this.$router.push({ name: 'Files' });
       }
     },
+    // eslint-disable-next-line no-unused-vars
+    acceptAlert(color) {
```
Comment (shakyaprasen) [inline src/views/Upload.vue:69]: can you create a utility function for this which can be used across components

File: src/views/Test.vue (hunk 1 above)
Comment (Drishtantr) [reply src/views/Test.vue]: done

File: src/views/Test.vue (hunk 2 above)
Comment (Drishtantr) [reply src/views/Test.vue]: done

File: src/views/Upload.vue (hunk 4 above)
Comment (Drishtantr) [reply src/views/Upload.vue]: done

**Pull request merged**